README.md

```diff
@@ -49,16 +49,18 @@ If you have any legacy android support library usage, the task will fail and pri
 Project: app
 ========================================
 
-Scanning com.github.bumptech.glide:glide:3.8.0
- * com/bumptech/glide/Glide.class -> android/support/v4/app/FragmentActivity
- * com/bumptech/glide/Glide.class -> android/support/v4/app/Fragment
- * com/bumptech/glide/manager/RequestManagerRetriever.class -> android/support/v4/app/FragmentManager
- * com/bumptech/glide/manager/RequestManagerRetriever.class -> android/support/v4/app/FragmentActivity
- * com/bumptech/glide/manager/RequestManagerRetriever.class -> android/support/v4/app/Fragment
- * com/bumptech/glide/manager/SupportRequestManagerFragment.class -> android/support/v4/app/Fragment
-
-Legacy support dependencies:
- * com.android.support:support-annotations:28.0.0
+Scanning com.squareup.rx.idler:rx2-idler:0.9.1
+ * com/squareup/rx2/idler/DelegatingIdlingResourceScheduler.class -> android/support/test/espresso/IdlingResource$ResourceCallback
+ * com/squareup/rx2/idler/DelegatingIdlingResourceScheduler.class -> android/support/test/espresso/IdlingResource
+ * com/squareup/rx2/idler/IdlingResourceScheduler.class -> android/support/test/espresso/IdlingResource
+ * com/squareup/rx2/idler/Rx2Idler$1.class -> android/support/test/espresso/IdlingResource
+ * com/squareup/rx2/idler/Rx2Idler$1.class -> android/support/test/espresso/Espresso
+ * pom -> com.android.support.test.espresso:espresso-core:2.2.2
+ * pom -> com.android.support:support-annotations:26.1.0
+
+Direct declarations of legacy support dependencies:
```

Comment:
Maybe this legend is better:

`Explicit declarations of legacy support dependencies on this project:`